src/Accounts/Authentication/Authentication/TokenCache/PowerShellTokenCacheProvider.cs

```diff
@@ -164,7 +165,7 @@ private SubscriptionClient GetSubscriptionClient(IAccessToken token, IAzureEnvir
 
         public virtual IPublicClientApplication CreatePublicClient(string authority = null)
         {
-            var builder = PublicClientApplicationBuilder.Create(PowerShellClientId);
+            var builder = PublicClientApplicationBuilder.Create(PowerShellClientId).WithBrokerPreview();
```

Comment:
I'm reading this page https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token-wam#wam-calling-pattern. Shall we use `.WithBroker()`?

Reply:
That was the old WAM API, which had issues like big assembly size (Windows SDK). [New applications should use WAM preview.](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/wam#new-wam-preview-in-msal-444)